tempesta_fw/t/functional/helpers/control.py

```diff
@@ -108,19 +108,71 @@ def set_user_agent(self, ua):
 class Wrk(Client):
     """ wrk - HTTP benchmark utility. """
 
+    class ScriptGenerator(object):
```

Comment:
Why you have decided to define class inside `Wrk` class? It's requred only in some cases, better to move the definition into separate file like it's done for `tempesta.py` and `nginx.py` which provide advanced control options for `Tempesta` and `Nginx` classes. I admit i'm not good in python and there can be much and much better implementations of current code, but i can't  agree here.